ed/plan.sh

```diff
@@ -1,12 +1,12 @@
-pkg_name=ed
+pkg_name="ed"
```

Comment:
This change is to appease shell check which has a warning for using unquoted literals that happen to match common commands since it might be indicative of a mistake: https://github.com/koalaman/shellcheck/blob/1ca0b72329b0687344c01302e0585a9e4e383502/src/ShellCheck/Data.hs#L70-L94